x-pack/plugins/uptime/public/hooks/use_filter_update.ts

```diff
@@ -8,46 +8,74 @@
 import { useEffect } from 'react';
 import { useUrlParams } from './use_url_params';
 
+export const parseFiltersMap = (currentFilters: string) => {
+  let filterKueries: Map<string, string[]>;
+  try {
+    filterKueries = new Map<string, string[]>(JSON.parse(currentFilters));
+  } catch {
+    filterKueries = new Map<string, string[]>();
```

Comment:
I guess this will result in us blanking out any invalid filter string in the URL, thoughts if it's a good way to go?